### PR TITLE
[474] Section "Kontakt z administratorem" accessible without login

### PR DESCRIPTION
**Pull request by mlipa**
__Story / Bug id:__ https://github.com/CodeForPoznan/volontulo/issues/474

__Reference person:__ @mlipa

__Description:__

Change template title.

### CODE REVIEW

File: apps/volontulo/lib/email.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     'offer_application': u'Zgłoszenie chęci pomocy w ofercie',
     'offer_creation': u'Zgłoszenie oferty na Volontulo',
     'registration': u'Rejestracja na Volontulo',
-    'volunteer_to_admin': u'Kontakt z administratorem',
+    'contact_to_admin': u'Kontakt z administratorem',
```
Comment (magul) [inline apps/volontulo/lib/email.py]: remove `u` prefix

File: apps/volontulo/templates/emails/contact_to_admin.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {% load staticfiles %}
 
-{% block title %}Kontakt wolontariusza z administratorem{% endblock %}
+{% block title %}Kontakt{% if applicant == 'VOLUNTEER' %} wolontariusza {% elif applicant == 'ORGANIZATION' %} organizacji {% endif %}z administratorem{% endblock %}
```
Comment (magul) [inline apps/volontulo/templates/emails/contact_to_admin.html]: move spaces outside of if-statement

**Comment by mlipa** [comment]
@magul 
Code review fixes applied.

**Comment by magul** [comment]
@mlipa could You correct pull request name and atach correct issue?

**Pull request merged**